cdc/processor/pipeline/table_actor.go

```diff
@@ -0,0 +1,400 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipeline
+
+import (
+	"context"
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"github.com/pingcap/tiflow/cdc/entry"
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/pingcap/tiflow/cdc/redo"
+	"github.com/pingcap/tiflow/cdc/sink"
+	"github.com/pingcap/tiflow/cdc/sink/common"
+	"github.com/pingcap/tiflow/pkg/actor"
+	"github.com/pingcap/tiflow/pkg/actor/message"
+	serverConfig "github.com/pingcap/tiflow/pkg/config"
+	cdcContext "github.com/pingcap/tiflow/pkg/context"
+	cerror "github.com/pingcap/tiflow/pkg/errors"
+	"github.com/pingcap/tiflow/pkg/pipeline"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+)
+
+var (
+	_ TablePipeline = (*tableActor)(nil)
+	_ actor.Actor   = (*tableActor)(nil)
+)
+
+type tableActor struct {
+	cancel    context.CancelFunc
+	wg        *errgroup.Group
+	reportErr func(error)
+
+	mb actor.Mailbox
+
+	changefeedID string
+	// quoted schema and table, used in metrics only
+	tableName     string
+	tableID       int64
+	markTableID   int64
+	cyclicEnabled bool
+	memoryQuota   uint64
+	mounter       entry.Mounter
+	replicaInfo   *model.TableReplicaInfo
+	sink          sink.Sink
+	targetTs      model.Ts
+
+	started bool
+	stopped bool
+	err     error
+
```

Comment:
I think `globalVars` would be a more self-explaining name.

Reply:
done